space_packet_parser/packets.py

```diff
@@ -245,21 +256,47 @@ def __init__(self, *args, raw_data: bytes = b"", **kwargs):
     @property
     def header(self) -> dict:
         """The header content of the packet."""
+        warnings.warn("The header property is deprecated and will be removed in a future release. "
+                      "To access the header fields of a CCSDS packet, use the CCSDSPacketBytes class.")
         return dict(list(self.items())[:7])
 
     @property
     def user_data(self) -> dict:
         """The user data content of the packet."""
+        warnings.warn("The user_data property is deprecated and will be removed in a future release. "
+                      "To access the user_data fields of a CCSDS packet, use the CCSDSPacketBytes class.")
         return dict(list(self.items())[7:])
 
+class CCSDSPacket(Packet):
+    """Packet representing parsed data items from CCSDS packet(s). DEPRECATED
+
+    This class is deprecated and will be removed in a future release. Use the Packet class instead.
+    In an XTCE representation, there is no guarantee that the CCSDS packet header will be defined
+    as individual elements. If you want to access those elements, you can use the CCSDSPacketBytes
+    class to extract the header fields with specific methods.
+
+    Container that stores the raw packet data (bytes) as an instance attribute and the parsed
+    data items in a dictionary interface. A ``CCSDSPacket`` generally begins as an empty dictionary that gets
+    filled as the packet is parsed. The first 7 items in the dictionary make up the
+    packet header (accessed with ``CCSDSPacket.header``), and the rest of the items
+    make up the user data (accessed with ``CCSDSPacket.user_data``). To access the
+    raw bytes of the packet, use the ``CCSDSPacket.raw_data`` attribute.
+    """
+    def __init__(self, *args, **kwargs):
+        warnings.warn("The CCSDSPacket class is deprecated and will be removed in a future release. "
```

Comment:
Make this an explicit `DeprecationWarning`. See https://docs.python.org/3/library/warnings.html#warnings.warn
The `stacklevel` kwarg can kick the warning up the stack so that it refers the user to the code that is _calling_ the deprecated code rather than pointing to the deprecated code itself.

Reply:
Yeah, I generally don't use DeprecationWarning because it is hidden by default unfortunately. I can add the stacklevel here to boost them.